### DISCUSSION TITLE
FastSDCPU ComfyUI extension

### DISCUSSION THREAD

**Discussion by rupeshs** [rupeshs/fastsdcpu#259] [Announcements]
Now it is possible to use FastSDCPU with ComfyUI, thanks [BetaDoggo](https://github.com/BetaDoggo) for creating this extension
Extension repo : https://github.com/BetaDoggo/ComfyUI-FastSDCPU
![fastsd-comfyui](https://github.com/user-attachments/assets/054b24e6-a470-4635-81e7-647dc5573555)

**Comment by sfrac** [comment]
If you're going to be using NPU from the new intel core series:  
1.   .\start-webserver.bat --use_openvino
2.   Choose "rupeshs/sd15-lcm-square-openvino-int8" as your Vino Model
